BuildResidentialHPXML/measure.rb

```diff
@@ -1382,6 +1382,72 @@ def arguments(model) # rubocop:disable Lint/UnusedMethodArgument
     arg.setUnits('W')
     args << arg
 
+    perf_data_capacity_type_choices = OpenStudio::StringVector.new
+    perf_data_capacity_type_choices << 'Capacity'
+    # perf_data_capacity_type_choices << 'CapacityFractionOfNominal'
+
+    arg = OpenStudio::Measure::OSArgument.makeChoiceArgument('perf_data_capacity_type', perf_data_capacity_type_choices, false)
+    arg.setDisplayName('Detailed Performance Data: Capacity Type')
+    arg.setDescription('Capacity type of detailed performance data.')
+    arg.setUnits('Capacity')
+    args << arg
+
+    arg = OpenStudio::Measure::OSArgument.makeStringArgument('heating_perf_data_outdoor_temperatures', false)
+    arg.setDisplayName('Detailed Performance Data: Heating Outdoor Temperatures')
+    arg.setDescription('Outdoor temperatures of heating detailed performance data. If multiple performance data points, use a comma-separated list.')
+    arg.setUnits('deg-F')
+    args << arg
+
+    arg = OpenStudio::Measure::OSArgument.makeStringArgument('heating_perf_data_minimum_capacities', false)
+    arg.setDisplayName('Detailed Performance Data: Heating Minimum Capacities')
+    arg.setDescription('Minimum capacities of heating detailed performance data. If multiple performance data points, use a comma-separated list.')
```

Comment:
Need to be careful with terminology, _especially for COPs_. What you call "minimum COPs" is really "COPs at minimum speed" and they will have **larger** values than the "maximum COPs", which is really "COPs at maximum speed".

So make changes to all arguments as follows:
- "Heating Minimum Foos" => Heating Foos at Minimum Speed
"Minimum foos of heating detailed performance data." => "Foos at minimum speed of heating detailed performance data."